app/models/package.rb

```diff
@@ -111,6 +111,10 @@ def add_location(location_id)
     end
   end
 
+  def update_allow_web_publish
```

Comment:
I'm a bit confused here... function says `update_allow_web_publish` but we are setting the attribute to false?